examples/test_suite_virtual/test_suite_teardown.py

```diff
@@ -19,21 +19,21 @@
 from pykiso.auxiliaries import aux_virtual, aux_udp  # isort:skip
 
 
-@pykiso.define_test_parameters(suite_id=6, aux_list=[aux_udp], teardown_timeout=2)
+@pykiso.define_test_parameters(suite_id=6, aux_list=[aux_udp])
 class ReportFailedDuringTearDown(pykiso.BasicTestSuiteTeardown):
```

Comment:
GreyTest

Reply:
All those Grey/BasicTest mistakes should be fix now